gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryTile } from '../../gallery-components/GalleryTiles';
+import CodeExample from '../../CodeExample';
+
+import {
+  NxTable,
+  NxTableBody,
+  NxTableCell,
+  NxTableHead,
+  NxTableRow
+} from '@sonatype/react-shared-components';
+
+import NxTabsSimpleExample from './NxTabsSimpleExample';
+
+const tabsSimpleExampleCode = require('!!raw-loader!./NxTabsSimpleExample').default;
+
+export default function NxTabsPage() {
+  return (
+    <>
+      <GalleryDescriptionTile>
+        <h3 className="nx-h3">NxTabs</h3>
```

Comment:
I suspect each of the component description sections should use the `nx-tile-subsection` styles (see the nx-tile page for examples).  Additionally a summary paragraph before the first subsection would probably be good

Reply:
6fe2aeb